cmd/humioctl/alerts_install.go

```diff
@@ -86,6 +85,7 @@ Use the --force flag to update existing alerts with conflicting names.
 	cmd.Flags().StringVar(&filePath, "file", "", "The local file path to the alert to install.")
 	cmd.Flags().StringVar(&url, "url", "", "A URL to fetch the alert file from.")
 	cmd.Flags().StringVarP(&name, "name", "n", "", "Install the alert under a specific name, ignoring the `name` attribute in the alert file.")
+	cmd.Flags().StringVar(&runAsUserID, "runAsUserId", "", "Install the alert on behalf on another user. Requires the 'ChangeTriggersToRunAsOtherUsers' permission.")
```

Comment:
For consistency, I believe this should be 
```suggestion
	cmd.Flags().StringVar(&runAsUserID, "run-as-user-id", "", "Install the alert on behalf on another user. Requires the 'ChangeTriggersToRunAsOtherUsers' permission.")
```